### ISSUE TITLE
Project/Solution Creation: Reference the projects instead of their assemblies, where possible

### ISSUE THREAD

**Issue by MeFisto94** [MeFisto94/Papier#1]
With the addition of multi-assembly solutions (.sln), the current implementation only adds the source sets of the current assembly to their csproj, adding every other possible assembly as a DLL reference.
If you however now have projects A and B depending on each other, the users wants the IDE to understand that the projects come from the solution and not some source DLL where we got the decompilation from.

TLDR: After csprojs are created, try to replace DLL References by project references, if possible.
